src/Cell.tsx

```diff
@@ -133,21 +134,34 @@ function Cell<R>({
     style.transform = `translateX(${scrollLeft}px)`;
   }
 
+  if (!children) {
+    const formatterProps = {
+      value: rowData[column.key],
+      column,
+      rowIdx,
+      row: rowData,
+      isRowSelected,
+      onRowSelectionChange,
+      isSummaryRow
+    };
+    const { formatter } = column;
+
+    if (isValidElementType(formatter)) {
+      children = createElement<typeof formatterProps>(formatter, formatterProps);
+    }
+
+    if (isElement(formatter)) {
+      children = cloneElement(formatter, formatterProps);
```

Comment:
Let's remove the `React.ReactElement` type from `column.formatter` now.

Reply:
Done